CogniteSdk/test/fsharp/Relationships.fs

```diff
@@ -86,3 +86,433 @@ let ``Create and delete Relationships is Ok`` () = task {
     // Assert
     test <@ resExternalId = externalId @>
 }
+
+[<Fact>]
+let ``BETA: Relationships resource available`` () = task {
+    // Arrange
+    let filter = Beta.RelationshipFilter()
+    let query = Beta.RelationshipQuery(Filter=filter, Limit=Nullable 1)
+    // Act
+    let! res = writeClient.Beta.Relationships.ListAsync query
+
+    let len = Seq.length res.Items
+
+    // Assert
+    test <@ len <= 1 @>
+}
+
+[<Fact>]
+let ``BETA: create and delete relationships is ok`` () = task {
+    // Arrange
+    // NOTE: Requires the Label: "RelationshipsTestCreateAndDeleteLabel" to be defined
+    // Define Relationships to create
+    let externalId = Guid.NewGuid().ToString()
+    let label = "RelationshipsTestCreateAndDeleteLabel"
+    let relationshipCreateObject =
+        seq {
+            Beta.RelationshipCreate(
+                ExternalId=externalId,
+                SourceExternalId="RelationshipTestCreateAndDeleteSource",
+                SourceType="asset",
+                TargetExternalId="RelationshipTestCreateAndDeleteTarget",
+                TargetType="asset",
+                Confidence=Nullable 0.999F,
+                Labels=seq{CogniteExternalId(label)}
+            )
+        }
+
+    let listFilter =
+        Beta.RelationshipQuery(
+            Filter=Beta.RelationshipFilter(Labels=LabelFilter.Any(label))
+        )
+
+    // Act
+    let! createRes = writeClient.Beta.Relationships.CreateAsync relationshipCreateObject
+    let! getExtIds = writeClient.Beta.Relationships.ListAsync listFilter
+
+    // cleanup
+    let deleteIds = getExtIds.Items |> Seq.map (fun relationship -> relationship.ExternalId)
```

Comment:
Better way of doing tests it to test one function per test. (maybe not in f#?) Then if something breaks, you know how many things not functional.

Could we save the relationship to the variable and reuse it for delete test in the end of file?

Reply:
I doubt it, because the tests run in parallel we can't really guarantee that a "create" test will run prior to a "delete" test. For a delete test we need something to delete, so it implies that we first have to create a relationship. And unless we want to leave junk in our cdf project it's probably a good idea to delete the assets we create as well. 

We could move some of this behavior to setup/cleanup before and after our tests run, but in doing so we assume our code is correct prior to testing it.

Reply:
we could possibly run them sequentially? by using `[Collection("Sequential")]`
not sure about the rest, need to have a look more in dotnet testing features